src/main/resources/db/changelog/changes/12-add-fks-in-books_categories-table.yaml

```diff
@@ -0,0 +1,21 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-forign-keys-in-books-categories
```

Comment:
Don't need to create addForeignKeyConstraint here, you can add them in the create-books_categories-table changelog